microsoft-365/enterprise/ms-cloud-germany-transition-add-pre-work.md

```diff
@@ -24,69 +24,104 @@ description: "Summary: Pre-work when moving from Microsoft Cloud Germany (Micros
+- If you 're using **Skype for Business Online**, do [this stpe](#Skype-for-Business-Online)
+- If you're using a third-party mobile device management (MDM) solution, do [this step](#mobile-device-management).
+- If you're using **third-party services** or **line-of-business (LOB) apps** that are integrated with Office 365, do [this step](#line-of-business-apps).
+- If you're also using **Dynamics 365**, do [this step](#dynamics365).
+- If you're also using **Power BI**, do [this step](#power-bi).
+- If you're also using **Azure services** with your Office 365 subscription, do [this step](#microsoft-azure).
+
+## General tenant migration considerations
+
+**Applies to:** All customers using Office 365 in the Microsoft Deutschland Cloud instance
+
+Office 365 tenant and user identifiers are preserved during migration. Azure AD service calls are redirected from Microsoft Cloud Deutschland to Office 365 Global services and are transparent to Office 365 services.
+
+- General Data Protection Regulation (GDPR) Data Subject Requests (DSRs) are executed from the Azure Admin portal for future requests. Any legacy or non-customer diagnostic data that is resident in Microsoft Cloud Deutschland is deleted at or before 30 days elapse.
+- Multi-factor authentication (MFA) requests that use Microsoft Authenticator display as the user ObjectID (a GUID) while the tenant is copied to Office 365 services. MFA requests will perform as expected despite this display behavior.  Microsoft Authenticator accounts that were activated by using Office 365 services endpoints will display the user principal name (UPN).  Accounts added by using Microsoft Cloud Deutschland endpoints will display the user ObjectID but will work with both Microsoft Cloud Deutschland and Office 365 services endpoints.
 
 | Step(s) | Description | Impact |
 |:-------|:-------|:-------|
 | Prepare to notify users about restarting and signing in to and out of their clients after migration. | Office client licensing will transition from Microsoft Cloud Deutschland to Office 365 services in the migration. Clients pick up a new valid license after signing out of and in to Office clients. |	Users' Office products need to refresh licenses from Office 365 services. If licenses aren't refreshed, Office products may experience license validation errors. |
 | Ensure network connectivity to [Office 365 services URLs and IP addresses](https://aka.ms/o365urls). | All clients and services hosted by the customer that are used to access Office 365 service must be able to access the Office 365 Global services endpoints. <br>In case, you or your collaboration partners have firewall rules in place that would prevent accessing the URLs and IP addresses listed in [Office 365 services URLs and IP addresses](https://aka.ms/o365urls) must change the firewall rules to permit access to the Office 365 Global service endpoints| Failures of the service or client software can occur if this is not done before Phase 4  |
```

Comment:
```suggestion
| Ensure network connectivity to [Office 365 services URLs and IP addresses](https://aka.ms/o365urls). | All clients and services hosted by the customer that are used to access Office 365 service must be able to access the Office 365 Global services endpoints. <br>In case you or your collaboration partners have firewall rules in place that would prevent accessing the URLs and IP addresses listed in [Office 365 services URLs and IP addresses](https://aka.ms/o365urls) must change the firewall rules to permit access to the Office 365 Global service endpoints| Failures of the service or client software can occur if this is not done before Phase 4  |
```